docs/_source/getting_started/installation/configurations/user_management.md

```diff
@@ -42,99 +41,104 @@ rg.init(workspace="my_shared_workspace")
 rg.set_workspace("my_private_workspace")
 ```
 
-### *users.yml*
-
-The above user management model is configured using a YAML file which server maintainers can define before launching a Argilla instance.
-This can be done when launching Argilla from Python or with the provided `docker-compose.yml`. Read below for more details on the different options.
-
 ## Default user
 
-By default, if you don't configure a `users.yml` file, your Argilla instance is pre-configured with the following default user:
+By default, if the Argilla instance has no users, the following default admin user will be configured:
 
 - username: `argilla`
 - password: `1234`
 - api_key: `argilla.apikey`
 
-for security reasons we recommend changing at least the password and API key.
+For security reasons, we recommend changing at least the password and the API key. You can do this via the following CLI command or with the equivalent Python code:
+
+--> TBD
+the default user as follows:
+
+```bash
+python -m argilla.tasks.users.create_default --password newpassword --api-key new-api-key
+```
+
```

Comment:
The only way to change default user credentials is by creating a new user with username `argilla`. We can add python code snippet for general user creation

```suggestion
For security reasons, we recommend changing at least the password and the API key. You can do this via the following CLI command:
```bash
python -m argilla.tasks.users.create_default --password newpassword --api-key new-api-key
```
```